nand.py

```diff
@@ -30,9 +33,10 @@ def write(self, address, wdata):
         with open(self.data_file, 'r', encoding='utf-8') as f:
             lines = f.readlines()
 
-        if address <= len(lines):
-            lines[address] = wdata + '\n'
-            with open(self.data_file, 'w', encoding='utf-8') as f:
-                f.writelines(lines)
-        else:
+        if address > len(lines):
```

Comment:
```suggestion
        if address >= len(lines):
```
이거 혹시 등호가 붙어야하는거 아닌가 생각이 듭니다.

예를들어 len(lines)==1이고 address == 1이면
27line에 lines[address] 가 index out of range 가 발생할것같습니다.